src/cmd/script.rs

```diff
@@ -0,0 +1,62 @@
+use anyhow::{Error, Result};
+use clap::Parser;
+use move_binary_format::file_format::CompiledScript;
+use std::fs;
+use std::path::PathBuf;
+
+use super::script_transaction::ScriptTransaction;
+use crate::run_context::RunContext;
+
+/// Create a script transaction.
+#[derive(Parser, Debug)]
+#[clap(about = "smove create-transaction")]
+pub struct CreateTransaction {
+    #[clap(short, long, help = "Path for the compiled Move script.")]
+    compiled_script_path: PathBuf,
+    // TODO(rqnsom): add script parameters here.
+}
+
+impl CreateTransaction {
+    /// Executes the command.
+    pub fn execute(&mut self, ctx: &RunContext) -> Result<()> {
+        let compiled_script = &self.compiled_script_path;
+        let script_bc = fs::read(compiled_script)
+            .map_err(|e| Error::msg(format!("Can't read '{}':\n{e}", compiled_script.display())))?;
+
+        // Check the script bytecode and verify the parameter rules.
+        verify_script_integrity(&script_bc)?;
+
+        // TODO(Rqnsom): maybe use a function to create this:
+        let tx = ScriptTransaction {
+            bytecode: script_bc,
+            args: vec![],
+            type_args: vec![],
+        };
+
+        // Path to the output file.
+        let tx_name = compiled_script.file_name().unwrap_or_default();
```

Comment:
What would be the default?
Does it make sense here to use default?

Reply:
Yeah, actually, it makes no sense.
(From discussion on Slack):
This can't ever fail in case the line 23 succeeds.
So let's use an `unwrap` with a comment here

Reply:
Done